### ISSUE TITLE
[SOLVED] Assets not found :(

### ISSUE THREAD

**Issue by leandroramos** [FlorientR/laravel-gentelella#54]
Hi,
I don't know why, but the assets can't be found after installing (following readme.md) and running the application, I see the errors:

/css/font-awesome.min.css - No such file or directory
/css/bootstrap.min.css - No such file or directory
/css/gentelella.min.css - No such file or directory
/js/jquery.min.js - No such file or directory
/js/bootstrap.min.js - No such file or directory
/js/gentelella.min.js - No such file or directory

Thank you

**Comment by FlorientR** [comment]
Hi, 

Have you run gulp ?

**Comment by leandroramos** [comment]
Yes, I did, but it didn't returned anything.

**Comment by vvp242424** [comment]
@leandroramos you have to make sure you run    npm install --global bower gulp      I ran into the same thing as you then after the project was ready I re ran npn install --global bower gulp  then right after words I ran just gulp and it fetched the proper files returning with 

Fetching Copy Source Files...
   - vendor/bower_components/gentelella/vendors/bootstrap/fonts//**/*

So the steps I followed were 
Step 1
With Composer

`composer create-project florientr/laravel-gentelella MyProject`

Step 2

```
npm install --global bower gulp
npm install gulp
bower install
gulp
```

Step 3 
`cp .env.example .env`

then configured my .env

`php artisan migrate`

That worked for me.

**Comment by leandroramos** [comment]
Thank you @vvp242424 

Fetching Copy Source Files...
   - vendor/bower_components/gentelella/vendors/font-awesome/fonts//**/*

Gulp worked for me too, the files are in the folders, but laravel still doesn't find the assets :(

"/usr/bin/php" "-S" "localhost:8002" "server.php"
[Mon Jul  3 09:46:10 2017] ::1:44468 [404]: /css/bootstrap.min.css - No such file or directory
[Mon Jul  3 09:46:10 2017] ::1:44470 [404]: /css/font-awesome.min.css - No such file or directory
[Mon Jul  3 09:46:10 2017] ::1:44472 [404]: /css/gentelella.min.css - No such file or directory
[Mon Jul  3 09:46:10 2017] ::1:44474 [404]: /favicon.ico - No such file or directory

**Comment by vvp242424** [comment]
@leandroramos have you tried to create it from scratch on another computer or a vm? I ended up changing directories and trying a new fresh install to get mine to work. I run mac with a parallels VM of mac to do testing and dev. The warnings are from them requiring older versions of the packages. shouldn't affect the install from working. I ended up going with another dashboard theme from themeforest instead of this one. Sorry couldn't help more.
